cms/core/utils.py

```diff
@@ -43,3 +43,11 @@ def get_formatted_pages_list(
             }
         data.append(datum)
     return data
+
+
+def get_client_ip(request: "HttpRequest") -> str | None:
```

Comment:
I don't think we should be logging `IP addresses`: https://github.com/ONSdigital/dp-standards/blob/main/LOGGING_STANDARDS.md#sensitive-information.

This functionality may need to be removed, and then this test would not be needed.

@MebinAbraham

Reply:
Realised @MebinAbraham  pointed this out too https://github.com/ONSdigital/dis-wagtail/pull/135#discussion_r2022481984

Reply:
Resolving based on: https://github.com/ONSdigital/dis-wagtail/pull/135#discussion_r2024503727